Pidaq/Lib/uploadToDrive/uploadToDrive.py

```diff
@@ -0,0 +1,118 @@
+from __future__ import print_function

```

Comment:
I don't think this file is necessary anymore, you can remove it from git using `git rm uploadToDrive.py` and then committing that